src/components/Slider/__stories__/Slider.stories.mdx

```diff
@@ -0,0 +1,219 @@
+import {ArgsTable, Canvas, Meta, Story} from "@storybook/addon-docs";
+import { ComponentRules, createComponentTemplate, createStoryMetaSettings, RelatedComponents, VerticalStories } from "../../../storybook";
+import "./Slider.stories.scss";
+import Slider from "../Slider";
+import Sound from "../../Icon/Icons/components/Sound";
+import Video from "../../Icon/Icons/components/Video";
+import Chips from "../../Chips/Chips";
+
+export const argTypes = createStoryMetaSettings({
+  component: Slider,
+  enumPropNamesArray: ["color", "size"]
+  // iconPropNamesArray: ["leftIcon", "rightIcon", "successIcon"]
+});
+
+<Meta title="Components/Slider/Slider" component={Slider} argTypes={argTypes} />
+
+<!--- Component template -->
+
+export const sliderTemplate = createComponentTemplate(Slider);
+
+<!--- Component documentation -->
+
+# Slider
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+
+Please write here the component description
+
+<Canvas>
+  <Story name="Overview" args={{}}>
+    {sliderTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable of={Slider} />
+
+## Usage
+
+<UsageGuidelines guidelines={["Usage guideline 1", "Usage guideline 2", "Usage guideline 3"]} />
```

Comment:
Please check with Evigny if he have any usage and tip to insert to the component story. if he does not have any we can remove this section.

Reply:
Yes we are working on it.
Part of things already done.
Issue can be resolved.